plugins/inputs/prometheus/prometheus.go

```diff
@@ -43,6 +44,9 @@ type Prometheus struct {
 	Username string `toml:"username"`
 	Password string `toml:"password"`
 
+	// Headers map
+	Headers map[string]string
```

Comment:
Can you please add `toml:"headers"` here.

Reply:
done